vstgui/lib/cframe.h

```diff
@@ -7,6 +7,7 @@
 
 #include "vstguifwd.h"
 #include "cviewcontainer.h"
+#include "cdrawcontext.h"
```

Comment:
remove this include by adding CBitmapInterpolationQuality into the vstguifwd.h header, thanks.